### PR TITLE
Add a comment in the dialog that appears when running the Dart Remote debugger

### PR DESCRIPTION
**Pull request by jwren**
@devoncarew @alexander-doroshko 

The label above the input field doesn't wrap (or take HTML), but we are able to add a "comment" which shows up as the light gray text above OK & Cancel:

Screenshot from 2020-02-13 13-27-28

### CODE REVIEW

File: Dart/resources/messages/DartBundle.properties [hunk 1]
```diff
@@ -226,6 +226,8 @@ warning.no.projects.selected.dart.support.will.be.disabled=Warning\: no projects
 debugger.trying.to.connect=Trying to connect
 enter.url.to.running.dart.app=Enter a URL to a running Dart or Flutter application
 connect.to.running.app.title=Connect to a Running App
+connect.to.running.app.comment=Running Dart processes have a VM service that can be connected to via a URL, the URL to connect to the VM \
```
Comment (devoncarew) [inline Dart/resources/messages/DartBundle.properties]: I'd avoid mentioning the Observatory specifically.

I believe the text is a run-on sentence; I'd re-work into multiple sentence, use a semi-colon, or similar.

**Comment by jwren** [comment]
Screenshot from 2020-02-13 14-00-25

**Comment by alexander-doroshko** [comment]
Thanks, merged!

**Comment by alexander-doroshko** [comment]
Just noticed that according to our [guidelines](https://jetbrains.github.io/ui/controls/input_field/), label shouldn't start with `Enter`. I changed it to 
```
VM debugger service URL of a running Dart or Flutter application:
```
Does it look good?

**Comment by jwren** [comment]
No objection from me, they seem equivalent.  @jacob314  -- any objection to having me change the label in DevTools?

**Comment by alexander-doroshko** [comment]
Or maybe `debugger` -> `debug`? (I noticed a screenshot by @devoncarew in https://github.com/flutter/flutter-intellij/pull/4381).
```
VM debug service URL of a running Dart or Flutter application:
```
BTW do we need `VM`?
```
Debug service URL of a running Dart or Flutter application:
```

**Comment by jwren** [comment]
I think you are right, `VM` is implied by the "running Dart or ... " Part of the problem here is that there isn't a formal name for this URL.

I'm fine with

`Debug service URL of a running Dart or Flutter application:`

**Pull request closed without merging**